pkg/proc/types.go

```diff
@@ -549,11 +557,15 @@ func nameOfStructRuntimeType(_type *Variable, kind, tflag int64) (string, error)
 				if err != nil {
 					return "", err
 				}
+
+			case "offsetAnon":
+				offsetAnon, _ := constant.Int64Val(field.Children[i].Value)
+				isembed = offsetAnon%2 != 0
```

Comment:
it's just a 2, it's used to determine if the least significant bit of offsetAnon is 1 or 0. I've added a comment explaining why here.